main_app/services.py

```diff
@@ -0,0 +1,9 @@
+from django.http import JsonResponse
+import requests
+
+def get_parks(self, pk):
+    info = requests.get('https://jsonplaceholder.typicode.com/todos/1').json()
+    data = {
+    'name': info['title']
+    }
+    return JsonResponse(data)
```

Comment:
We'll likely need to pull some of this out and put it into a Serializer class. Just not sure how to do that yet.

Reply:
Looks great!